pages/research_software/three_tier_view.md

```diff
@@ -4,16 +4,10 @@ title: Three-Tier Model of Research Software
 
 The **three-tier model of research software** provides a framework for understanding the diverse landscape of software 
 in research - from scripts, code, notebooks, to computational workflows, libraries, modules, frameworks, utilities and applications. 
-This model distinguishes three tiers of research software, focussing on what they get built for:
 
-- **Analysis code** - research software that captures computational research processes and methodology, often used in simulation, 
-data generation, preparation, analysis and visualisation. It typically represents software created for personal use with a small scope, such as analysis scripts. 
-- **Prototype tools** - research software that demonstrates a new idea, method or model for use beyond the project in which it originated,
-  often as a substantive intellectual contribution or a proof of concept.
-  These software tools are designed to answer multiple research questions and are typically developed and used by more than one person.
-- **Research software infrastructure** - software that captures broadly accepted ideas, methods and models for use in research,
-  warranting close researcher involvement in their development.
-  This tier represents a broadly applicable research software, often with a large and possibly distributed development team.
+The three-tier model acknowledges a relationship between the tiers: software at higher tiers builds on the existence and stability of software in lower tiers,
```

Comment:
That's not true - the analysis script might be build on a library which is considered infrastructure - I would get rid of this line and the following two lines also - unless they came from any reference docs.